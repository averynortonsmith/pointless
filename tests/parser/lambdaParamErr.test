
x = (123) => foo