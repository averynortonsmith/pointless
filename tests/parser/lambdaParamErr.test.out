-------------------------------------------------------------------------------
Parser Error:

Expected 'Newline or Semicolon', got 'Lambda'
-------------------------------------------------------------------------------

At (line 2 column 11) in 'tests/parser/lambdaParamErr.test'
x = (123) => foo
          ^
